eng/Generation.psm1

```diff
@@ -93,42 +93,17 @@ function Invoke-CadlSetup()
     $emitterPath = Join-Path $PSScriptRoot ".." "src" "CADL.Extension" "Emitter.Csharp"
     $emitterPath = Resolve-Path -Path $emitterPath
     Push-Location $emitterPath
-    Try 
-    {
-        npm ci
-        npm run build
-        npm pack
-    }
-    Finally 
-    {
-        Pop-Location
-    }
 
-    # install cadl and emitter
-    $repoRoot = Join-Path $PSScriptRoot ".."
-    $repoRoot = Resolve-Path $repoRoot
-    Push-Location $repoRoot
     Try 
     {
-        npm ci
-        $packages = Get-ChildItem $repoRoot -Filter azure-tools-cadl-csharp-*.tgz -Recurse | Select-Object -ExpandProperty FullName | Resolve-Path -Relative
-        if ($packages) 
-        {
-            $package = $packages;
-            if ($packages.count -gt 1)
-            {
-                $package = $packages[0]
-            }
-            npm install $package
-        }
-        git checkout $repoRoot/package.json
-        git checkout $repoRoot/package-lock.json
+        npm run build
```

Comment:
Will it use the emitter version build locally?

Reply:
You mean in CI? It will use the build in the build agent.